### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by shubham-pampattiwar**
- updated readme with noobaa doc links
- added new scenarios in noobaa debug doc

### CODE REVIEW

File: deploy/crds/konveyor.openshift.io_v1alpha1_velero_cr.yaml [hunk 1]
```diff
@@ -1,17 +1,18 @@
-apiVersion: konveyor.openshift.io/v1alpha1
+zxapiVersion: konveyor.openshift.io/v1alpha1
```
Comment (dymurray) [inline deploy/crds/konveyor.openshift.io_v1alpha1_velero_cr.yaml:1]: Typo

File: deploy/crds/konveyor.openshift.io_v1alpha1_velero_cr.yaml [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
+  use_upstream_images: true
   default_velero_plugins:
   - aws
   - openshift
   backup_storage_locations:
   - name: default
     provider: aws
     object_storage:
-      bucket: myBucket
+      bucket: shubbam-6109f5e9711c8c58131acdd2f490f451
```
Comment (dymurray) [inline deploy/crds/konveyor.openshift.io_v1alpha1_velero_cr.yaml:15]: Please revert this

File: deploy/crds/konveyor.openshift.io_v1alpha1_velero_cr.yaml (hunk 1 above)
Comment (shubham-pampattiwar) [reply deploy/crds/konveyor.openshift.io_v1alpha1_velero_cr.yaml]: Done!

File: deploy/crds/konveyor.openshift.io_v1alpha1_velero_cr.yaml (hunk 2 above)
Comment (shubham-pampattiwar) [reply deploy/crds/konveyor.openshift.io_v1alpha1_velero_cr.yaml]: Updated PR

**Pull request merged**